docs/NooBaaNonContainerized/Logging.md

```diff
@@ -19,10 +19,19 @@ This document provides an overview of the logging capabilities within NooBaa, de
 
 ## NooBaa Service Logs
 
-NooBaa service logs are written directly to stderr and to syslog -  
-1. NooBaa writes the logs into stderr and to `/var/log/noobaa.log`.
-2. The `journalctl` service watches stderr and writes it to `journal`.
-3. A `syslog` service, like `rsyslog/syslog_ng` watches `journal` and writes it to `/var/log/messages/`.
+NooBaa service logs are written directly to stderr, syslog, or both.
+1. The `journalctl` service watches stderr and writes into `journal`. That internally updates the `/var/log/message`.
+2. Noobaa sends logs to a syslog service such as rsyslog or syslog_ng and writes the logs to `/var/log/noobaa.log` if enabled. We can not have more than one service enabled at a time. It might cause some issues.
+
+### Set log config
+
+Set `LOG_TO_STDERR_ENABLED` property to `true` to  send Noobaa logs to stderr. Noobaa logs will be send to syslog when the property `LOG_TO_SYSLOG_ENABLED` set to `true`.
+
+Default configurations are:
+```
+LOG_TO_STDERR_ENABLED = true;
+LOG_TO_SYSLOG_ENABLED = false;
+```
 
 ### Journal logs
 1. NooBaa service logs -
```

Comment:
can you also add here a way to fix `Suppressed 6284127 messages from noobaa.service`
RateLimitInterval=0
RateLimitBurst=0
you can attach a link to the fix mentioned in this link - https://my.f5.com/manage/s/article/K70501143

Reply:
@romayalon we already have this in our code? de we really need to mention it again?
https://github.com/noobaa/noobaa-core/blob/master/src/deploy/standalone/noobaa_syslog.conf

Reply:
we have it on the code for syslog conf, we experienced this issue on journal (stderr) for several times

Reply:
Ok will add it for journal,